# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libsigc++1/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur libsigc++-1.0.4-orig/tests/signals/convert_slot_test.cc libsigc++-1.0.4/tests/signals/convert_slot_test.cc
--- libsigc++-1.0.4-orig/tests/signals/convert_slot_test.cc	2008-07-23 03:55:18.000000000 +0000
+++ libsigc++-1.0.4/tests/signals/convert_slot_test.cc	2008-07-23 03:56:09.000000000 +0000
@@ -1,4 +1,5 @@
 #include <string>
+#include <cstring>
 #include <iostream>
 #include <sigc++/signal_system.h>
 #include <sigc++/convert.h>
